tools/cmd.go

```diff
@@ -67,3 +70,33 @@ func mustGetDuration(cmd *cobra.Command, flagName string) time.Duration {
 	}
 	return val
 }
+
+func ResolveManifestFile(input string) (manifestName string, err error) {
+	if input == "" {
+		_, err := os.Stat("substreams.yaml")
+		if err != nil {
+			if errors.Is(err, os.ErrNotExist) {
+				return "", fmt.Errorf("no manifest entered in dir w/o a manifest")
+			}
+			return "", fmt.Errorf("finding manifest: %w", err)
+		}
+
+		return "substreams.yaml", nil
+	}
+
+	inputInfo, err := os.Stat(input)
+	if err != nil {
+		return "", fmt.Errorf("reading input: %w", err)
+	}
+
+	// If input is dir
```

Comment:
I think the conditional `if inputInfo.IsDir() {` is obvious already.

Reply:
it didnt seem to be when i was testing the function. os.Stat was giving no error whether file or directory, which i also found weird so I added it

Reply:
I'm talking about the comment here, to remove it since `if inputInfo.IsDir() {` says the same as `// If input is dir`, sorry should have been clearer

Reply:
all good ill get rid of it